src/PID.c

```diff
@@ -0,0 +1,22 @@
+#include <main.h>
+
+/**
+ Default PID struct constructor, used for default parameter value
+ These specific values represent a smooth curve with no overshoot
+ */
+const struct PID DEFAULT_PID = { 5, 3, 3 };
+
+/**
+ Gets the output of the input passed through the PID controller for the delta time interval
+ */
+int pid(int input, int *errorSum, const unsigned long * const deltaTime, int target, PID pid) {
```

Comment:
`deltaTime` should either be a `const unsigned long` or a pointer to an `unsigned long`. It does not need to be a `const` pointer to an `const unsigned long`. `pid` should be a `const`.